ChangeLog.md

```diff
@@ -1,52 +1,5 @@
-# Pacemaker-3.0.0-rc3 (23 Dec 2024)
-* 33 commits with 9 files changed, 77 insertions(+), 12 deletions(-)
-
-## Features added since Pacemaker-3.0.0-rc2
-
-* Inkscape is no longer a build dependency for Pacemaker documentation
```

Comment:
What happened to this one?

Reply:
Anything in particular you are missing? PNGs are there, build dependency dropped from spec-file, ...

Reply:
@wenningerk This commit drops this changelog message.

Reply:
Usually build-stuff doesn't go into final changelog while it is interesting to track differnces between release-candidates. Mostly just stuff that changes the result ...
But you are right for this one it might make sense to mention.

Reply:
> Usually build-stuff doesn't go into final changelog while it is interesting to track differnces between release-candidates. Mostly just stuff that changes the result like the pacemaker_remoted link ...

This, especially since there are a number of other build-time changes that are summarized at the link at the top of the release notes.

> But you are right for this one it might make sense to mention.